km_api/know_me/admin.py

```diff
@@ -71,12 +71,15 @@ class KMUserAccessor(admin.ModelAdmin):
         'km_user',
         'user_with_access',
         'accepted',
-        'can_write_everywhere')
+        'can_write',
+        'has_private_profile_access')
```

Comment:
It occurred to me that these fields may be redundant. With the current model we were given, the admins are the only ones who can write, and they are the only ones with private profile access. What do you think?